oio/cli/common/clientmanager.py

```diff
@@ -60,6 +61,9 @@ def admin_mode(self):
             self._admin_mode = self._options.get('admin_mode')
         return self._admin_mode
 
+    def flatns_config(self, bitlength):
+        self._flatns_config['flat_bitlength'] = bitlength
```

Comment:
`flatns_set_bits` would be more readable. Or maybe: `flatns_set_options(self, **kwargs)`.